MIP/mip-40/README.md

```diff
@@ -0,0 +1,83 @@
+# MIP-40: *AB-FFS Decoupled*
+- **Description**: Introduces architectural guide for ***AB-FFS Decoupled***. 
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Reviewer**: [Andreas Penzkofer](mailto:andreas.penzkofer@movementlabs.xyz)
+- **Desiderata**: [MD-38](https://github.com/movementlabsxyz/MIP/pull/38)
+
+## Abstract
+
+This MIP introduces the architectural guide for implementing ***AB-FFS Decoupled***, a means of using the Atomic Bridge and Fast-finality Settlement with a fixed supply of token. This decoupled approach relies on burning gas on the L2 into an L2 pool and issuing rewards in an LP token on the L1. Details of the L2 pool recirculation and the L1 LP token issuance are largely left to follow-on MIPs, however we do specify general constraints and guidelines. 
```

Comment:
The bridge and FFS should not be coupled. 
There are two instances of MOVE tokens: L1MOVE and L2MOVE.
L1MOVE is the native token on L1, an ERC-20, and this is the currency in which validators are rewarded.
L2MOVE is the wrapped MOVE on L2. It is incidentally the token we use to charge gas, but it is not coupled to a Gas Token. I suggest looking at this PR https://github.com/movementlabsxyz/MIP/pull/39 to get an idea how a bridge looks like.

Reply:
The point of this, [MD-38](https://github.com/movementlabsxyz/MIP/pull/38), and subsequent proposals is that we have a supply issue which needs to be resolved to maintain a fixed supply. 

- Where does the L2MOVE go when it is used as gas?: This MIP says it goes back into a pool on the L2. 
- From where does FFS source its rewards?: This MIP says it is issued as an LP token. 

The reason it is called decoupled is because none of these systems are forced to rely on each other in this arrangement.